packages/fluentui/docs/src/vr-tests/Video/Video.stories.tsx

```diff
@@ -1,8 +1,15 @@
+import * as React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { ComponentMeta } from '@storybook/react';
-import { Video } from '@fluentui/react-northstar';
-import VideoExample from '../../examples/components/Video/Types/VideoExample.shorthand';
+import { Flex, Text, Video } from '@fluentui/react-northstar';
 
 export default { component: Video, title: 'Video' } as ComponentMeta<typeof Video>;
 
+const VideoExample = () => (
```

Comment:
created new `VideoExample` with no video source to prevent visual diff changes due to unpredictable loading time of video